python/lsst/pipe/tasks/measurePsf.py

```diff
@@ -69,24 +70,12 @@ class MeasurePsfTask(pipeBase.Task):
 
 \section pipe_tasks_measurePsf_Purpose	Description
 
-A task that wraps two algorithms set via a pair of registries specified in the task's
-\ref pipe_tasks_measurePsf_Config.
-Both algorithms are classes with a constructor taking a pex.config.Config object (\em e.g.
-lsst.meas.algorithms.objectSizeStarSelector.ObjectSizeStarSelector.__init__).
+A task that selects stars from a catalog of sources and uses those to measure the PSF.
 
-The algorithms are:
- - a star selector, a subclass of lsst.meas.algorithms.StarSelector.selectStars
-
- - a psf estimator with API:
-\code
-determinePsf(exposure, psfCandidateList, metadata=None, flagKey=None)
-\endcode
-which returns an lsst.afw.detection.Psf and lsst.afw.math.SpatialCellSet (\em e.g.
-lsst.meas.algorithms.pcaPsfDeterminer.PcaPsfDeterminer.determinePsf).
-MeasurePsfTask calls determinePsf with \c flagKey set to
-"calib.psf.used" if a schema is passed to its constructor (see \ref pipe_tasks_measurePsf_Initialize).
-
-See also lsst.meas.algorithms.psfDeterminerRegistry.psfDeterminerRegistry.
+The star selector is a subclass of
+\ref lsst.meas.algorithms.starSelector.StarSelectorTask "lsst.meas.algorithms.StarSelectorTask"
+and the PSF determiner is a sublcass of
+\ref lsst.meas.algorithms.psfDeterminer.PsfDeterminerTask "lsst.meas.algorithms.PsfDeterminerTask"
```

Comment:
Yes. I pushed that fix after I asked you to review it and am a bit worried you are still seeing the older version.